### PR TITLE
Create automation.md (PR automation)

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: automation.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Automation
+
+## Activated automations:
+
+ - **Out of the box github automerge** will merge PR once everything is green. It is activable from a pull request page.
```
Comment (ryestew) [inline automation.md:5]: A small edit:
**Out of the box, github automerge** will merge a PR once everything is green. It is activated from a pull request page.

File: automation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ 
+   It won't automatically sync (merge or rebase) the branch with master.    
+ 
+   For more [information](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
+ 
+ - **Autosquash** will automatically update the branch - merge commit (when a new commit lands on master).
+ 
+   It will automatically squash and merge to master once everything is green.
+   
+   It is activable by adding the `autosquash` label.
```
Comment (ryestew) [inline automation.md:15]: It is activated

**Review by ryestew — CHANGES_REQUESTED** [review]
Please change the word  **activable** to **activated** - see the specifics in my 2 comments

**Comment by yann300** [comment]
@ryestew yes that what I did .did I miss something ?

**Review by ryestew — APPROVED** [review]

**Pull request merged**